text/0107-construct-library-module-lifecycle.md

```diff
@@ -30,20 +30,27 @@ module:
 - Stage 2 - Developer Preview
 - Stage 3 - GA
 
+Each stage in this document describes 3 distinct sets of activities:
+
+* **Entry activities:** Actions that need to be taken before the module enters this stage. These can either be automatic, or done by the team. They will usually be performed once a module is deemed for stage maturation, and all its exit criteria are met.
+* **Ongoing activities:** These are activities performed during the time a module lies statically in the given stage (e.g bug fixes, new features...)
+* **Exit criteria:** A module can progress to the next stage only if its exit criteria are met. Note that this is required, but not sufficient. A module matures to the next stage only if its exit criteria are met, **and** was prioritized over other modules using the [prioritization framework](#Construct-prioritization-framework).
+
 ## Stage 0 - CFN Resources
 
 All construct library modules start in stage 0 when they are auto-generated from the CloudFormation resource
 specification. The goal of stage 0 is to make new CloudFormation resources/properties available to CDK customers as soon
 as possible, and create tracking artifacts that allow us to capture the data required to make prioritization decisions
 to add L2 resources in the future.
 
-- **Stage 0 activities:**
-  - **Automatically publish package** - A package for the module is autogenerated and published in the aws-cdk GitHub
-    repo
+### Entry activities
 
-  - **Automatically publish README** - The module has a rudimentary README doc that (this is an automated activity):
-    - links to the relevant CloudFormation docs
-    - includes the basic import statement
+  > Many actions here are actually performed automatically during our weekly release process.
+  > - Generation of the module L1 code when CFN introduces a new spec.
+  > - Generation of the initial module README.
+  > - Mark module as "experimental" in `package.json`.
+  >
+  > The remaining activities described here need to be done manually.
 
   - **Create GitHub tracking issue** - Create a GitHub tracking issue for the module. The primary role of the tracking
```

Comment:
Minor: indenting. Now that the stage names are headers, you can drop the 2 space indent and align all of them right at the margin.

Reply:
Cool